standards_lab/ui/templates/project.html

```diff
@@ -324,6 +335,24 @@ <h2 class="card-title">Test</h2>
         });
       },
 
+      /* Delete project */
+      deleteProject: function(){
+
+        fetch(ownedProjectApiUrl, {
+          method:'DELETE',
+          credentials: 'same-origin',
+          headers: { 'X-CSRFToken': csrfmiddlewaretoken_value },
+          body: JSON.stringify(this.project),
+        }).then(response => {
+            if (response.ok) {
+              window.location.href = "/";
```

Comment:
Could we do like you've done with ownedProjectsApiUrl and have this as `var standardsLabHomeUrl = {% url "ui:home" %}`

Reply:
Good suggestion